lib/utils/helpers.js

```diff
@@ -14,3 +14,5 @@ export const intersperse = (list, separator) => {
 export const getRandomInt = limit => Math.floor(Math.random() * limit)
 
 export const getElementFromSeed = (list, seed) => list[seed % list.length]
+
+export const findByKey = (list, key, value) => list.find(e => e[key] === value)
```

Comment:
This abstract increases the complexity of the code and make it harder to understand what is going on. Going to remove it.